monai/bundle/config_parser.py

```diff
@@ -463,3 +463,50 @@ def resolve_relative_ids(cls, id: str, value: str) -> str:
                 new = ID_SEP_KEY.join(current_id[:-length]) + ID_SEP_KEY
             value = value.replace(p, sym + new)
         return value
+
+    def convert_key_chain_to_id(self, key_chain: List[str]) -> str:
+        """
+        To simplify the use of ConfigParser in some scenario, this function can convert a list of string
+        keys (e.g. ["key1", "key2", "key3"]) to the ID format ("key1#key2#key3"), which can be used by
+        other functions
+
+        Args:
+            key_chain: a list of string keys
+
+        Return:
+            a formated string
+
+        """
+        id_str = ""
+        for key in key_chain:
+            id_str += key + ID_SEP_KEY
+        id_str = id_str[:-1]
+        return id_str
```

Comment:
I think this could be `ID_SEP_KEY.join(key_chain)`?

Reply:
Thanks for the suggestion. Improved in 47fe5f393de15cb6ee2acdf5e34f46dcf9b2a705.